spec/support/factory_bot.rb

```diff
@@ -11,6 +11,7 @@
   factory :giraffe do
     name { 'name' }
     description { 'description' }
+    image { 'image' } #need to use fixture_file upload
```

Comment:
Is this intentional?

Reply:
Im going to remove it because this is not even being used in our tests at the moment